### PR TITLE
added authorization credential manual

### PR DESCRIPTION
**Pull request by woutslakhorst**

### CODE REVIEW

File: mini-manuals/5-authz-credentials.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
+# Authorization credentials
+
+## Description
+
+This manual describes how to add and revoke authorization credentials.
+Authorization credentials are verifiable credentials that tell the subject where and which resources can be viewed (or changed).
+The credentials are issued by the custodian, the care organization that is responsible for the data that is to be shared.
+The credentials are issued to the actor, the care organization which users have a demand for patient data.
+
+Authorization credentials have a `purposeOfUse` which scope the credential to a specific (part of a) Bolt.
```
Comment (reinkrul) [inline mini-manuals/5-authz-credentials.md:10]: scopes

File: mini-manuals/5-authz-credentials.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+The [Nuts authorization credential RFC](https://nuts-foundation.gitbook.io/drafts/rfc/rfc014-authorization-credential) explains the details of the credential.
+For this manual we'll create two different authorization credentials. 
+The eOverdracht bolt requires similar credentials.
+
+Nuts authorization credentials are only synchronized between the issuer and the subject. 
+Other nodes will not be able to view the credential.
```
Comment (reinkrul) [inline mini-manuals/5-authz-credentials.md:16]: which is currently not true

File: mini-manuals/5-authz-credentials.md [hunk 3]
```diff
@@ -0,0 +1,159 @@
+# Authorization credentials
+
+## Description
+
+This manual describes how to add and revoke authorization credentials.
+Authorization credentials are verifiable credentials that tell the subject where and which resources can be viewed (or changed).
+The credentials are issued by the custodian, the care organization that is responsible for the data that is to be shared.
+The credentials are issued to the actor, the care organization which users have a demand for patient data.
```
Comment (reinkrul) [inline mini-manuals/5-authz-credentials.md:8]: do we also need to state something about the subject (patient)?

File: mini-manuals/5-authz-credentials.md [hunk 4]
```diff
@@ -0,0 +1,159 @@
+# Authorization credentials
+
+## Description
+
+This manual describes how to add and revoke authorization credentials.
+Authorization credentials are verifiable credentials that tell the subject where and which resources can be viewed (or changed).
+The credentials are issued by the custodian, the care organization that is responsible for the data that is to be shared.
```
Comment (reinkrul) [inline mini-manuals/5-authz-credentials.md:7]: "holder of the data"?

File: mini-manuals/5-authz-credentials.md [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+    }
+}
+```
+As you can see, there are quite some fields to fill out.
+The `issuer` is the DID of the custodian and the `credentialSubject.id` is the DID of the actor.
+The `type` must be a list of just `NutsAuthorizationCredential`.
+
+A bolt specifies how the `legalBase`, `purposeOfUse` and `resources` fields are filled.
+
+The example above describes an authorization credential that is created by the *sender* of the *eOverdracht* bolt.
```
Comment (reinkrul) [inline mini-manuals/5-authz-credentials.md:63]: sender of the *eOverdracht* transfer

File: mini-manuals/5-authz-credentials.md [hunk 6]
```diff
@@ -0,0 +1,159 @@
...
+The recipient of that credential is able to read and update the specified Task resource.
+No user authentication is required.
+It also allows the recipient to read and perform the FHIR *document* operation on the specified composition.
```
Comment (reinkrul) [inline mini-manuals/5-authz-credentials.md:66]: read -> *read*

File: mini-manuals/5-authz-credentials.md [hunk 7]
```diff
@@ -0,0 +1,159 @@
...
+Authorization credentials are only used in the request for an access token.
+Before you can use them, you'll have to find them first.
+We use the example above from the eOverdracht bolt.
+In that bolt, the *receiver* (the actor) has received a FHIR Task resource. 
+Within that resource a reference to a FHIR composition is listed.
+The receiver must find the authorization credential that authorizes access to that composition.
+The following call must be used to find it:
+
+```
+POST <internal-node-address>/internal/vcr/v1/authorization?untrusted=true
```
Comment (reinkrul) [inline mini-manuals/5-authz-credentials.md:82]: ouch, maybe we should provide some handy JSON Path-enabled API for this in the future for easier querying.

Comment (reinkrul) [inline mini-manuals/5-authz-credentials.md:82]: do we need to explain `untrusted=true`?

File: mini-manuals/5-authz-credentials.md [hunk 8]
```diff
@@ -0,0 +1,159 @@
+# Authorization credentials
+
+## Description
```
Comment (reinkrul) [inline mini-manuals/5-authz-credentials.md:3]: I'm missing something either in the Description section or at the bottom as conclusion, what the result is of this manual ("and now you can do XYZ, next follow manual ABC")

**Review by reinkrul — CHANGES_REQUESTED** [review]

File: mini-manuals/5-authz-credentials.md (hunk 2 above)
Comment (woutslakhorst) [reply mini-manuals/5-authz-credentials.md:16]: don't tell anyone

File: mini-manuals/5-authz-credentials.md (hunk 3 above)
Comment (woutslakhorst) [reply mini-manuals/5-authz-credentials.md:8]: we haven't implemented it in the demo, so we'll just skip it for now.

File: mini-manuals/5-authz-credentials.md (hunk 5 above)
Comment (woutslakhorst) [reply mini-manuals/5-authz-credentials.md:63]: I was actually referring to the role within the bolt

File: mini-manuals/5-authz-credentials.md (hunk 4 above)
Comment (woutslakhorst) [reply mini-manuals/5-authz-credentials.md:7]: made it even shorter.

File: mini-manuals/5-authz-credentials.md (hunk 7 above)
Comment (woutslakhorst) [reply mini-manuals/5-authz-credentials.md:82]:
```
{
    "credentialSubject.id": "did:nuts:JCJEi3waNGNhkmwVvFB3wdUsmDYPnTcZxYiWThZqgWKv"
}
```

would already be better.

Comment (woutslakhorst) [reply mini-manuals/5-authz-credentials.md:82]: we do this in the presentation

File: mini-manuals/5-authz-credentials.md (hunk 8 above)
Comment (woutslakhorst) [reply mini-manuals/5-authz-credentials.md:3]: I think we'll add that to the specific workshop READMEs. Start with 1,2,3 and maybe do 4

**Review by reinkrul — APPROVED** [review]

**Pull request merged**